cmflib/cmf.py

```diff
@@ -1365,6 +1375,128 @@ def update_model_output(self, artifact: mlpb.Artifact):
         """updates an artifact"""
         put_artifact(self.store, artifact)
 
+    def metadata_push(self, execution_id: str = ""):
+        """Pushes mlmd file to cmf-server """
+        # Required arguments:  pipeline_name, filename (mlmd file path) 
+        #Optional arguments: Execution_ID
+        output = mt_push(self.parent_context.name, self.filename, execution_id)
+        return output
+
+    def metadata_pull(self, execution_id: str = ""):
+        """Pulls mlmd file from cmf-server"""
+        # Required arguments:  pipeline_name, filename(file path to store mlmd file) 
+        #Optional arguments: Execution_ID
+        output = mt_pull(self.parent_context.name, self.filename, execution_id)
+        return output
+
+    def artifact_pull(self):
+        """Pulls artifacts from initialized repository """
+        # Required arguments: Pipeline_name
+        # Optional arguments: filename( path to store artifacts)
+        output = arti_pull(self.parent_context.name, self.filename)
+        return output
+
+    def artifact_push(self):
+        """Push artifacts to initialized repository"""
+        output = arti_push()
+        return output
+
+    def cmf_init_show(self):

```

Comment:
cmf init_show, init_* family of commands should be outside the cmf class, because they wont be able to create the class itself , if init is not done.

Reply:
Done